spec/example.spec.js

```diff
@@ -0,0 +1,15 @@
+describe('Test that the tests are running', function(){
```

Comment:
What is the plan for this? Will this file live on forever, or is it only an example? I think an example is a fine idea (it will give me something to build against for Travis), but add a TODO to remove it once the real code/tests are added.

Reply:
I was planning to delete it later, I'll add a TODO